Automattic/pocket-casts-ios
# Add a configuration to allow the display of the Toast above the mini-player

| 📘 Part of: # |  
|:---:|

Fixes # 


| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-12-16 at 14 20 59](https://github.com/user-attachments/assets/2d2118b0-b18f-4a8a-b618-dfc6dc0def28) | ![Simulator Screenshot - iPhone 16 Pro - 2024-12-16 at 14 20 49](https://github.com/user-attachments/assets/d5afa046-aaa9-411b-bfad-f8ef0d19d436) |

## To test

- Start the app with a user with a subscription
- Start playing an episode so the mini-player is visible
- Go to UpNext tab
- Activate UpNext Shuffle
- See if the Toast shows above the Mini-Player
- Tap on UpNext icon the mini-player
- Activate UpNext Shuffle
- - See if the Toast shows at the regular position

## Checklist
